jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/utils/PayloadCreator.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.homeoffice.digital.sas.jparest.utils;
+
+import java.util.List;
+
+public class PayloadCreator {
```

Comment:
As this is more of a utility class this class can be made final
```suggestion
public final class PayloadCreator {
```

Reply:
Done